packages/office-ui-fabric-react/src/components/Overlay/Overlay.tsx

```diff
@@ -1,41 +1,13 @@
-import * as React from 'react';
+import { styled } from '@uifabric/utilities';
```

Comment:
I think we wanted to do `import { styled } from '../../Utilities';`

Reply:
Nice catch.